projects/angular-draggable-droppable/src/lib/draggable.directive.ts

```diff
@@ -291,9 +291,11 @@ export class DraggableDirective implements OnInit, OnChanges, OnDestroy {
         const currentDrag$ = new Subject<CurrentDragData>();
         const cancelDrag$ = new ReplaySubject<void>();
 
-        this.zone.run(() => {
-          this.dragPointerDown.next({ x: 0, y: 0 });
-        });
+        if (this.dragPointerDown.observed) {
```

Comment:
Is `.observed` available in rxjs 6? Just worried that this might be a breaking change for those users as it's still supported by the latest angular. If it's not maybe we could switch it to the deprecated `.observers.length > 0` so as not to break backwards compatibility? Then when rxjs 8 comes we can do the upgrade and switch back with a breaking change. WDYT?

Reply:
Yeah that makes sense. `.observed` is available only in 7.1.0+ so I reverted back to `observers.length`.